src/sidebar/components/test/notebook-view-test.js

```diff
@@ -64,34 +64,8 @@ describe('NotebookView', () => {
     assert.equal(wrapper.find('.notebook-view__heading').text(), '…');
   });
 
-  describe('results count', () => {
-    [
-      {
-        rootThread: { totalChildren: 5, replyCount: 15 },
-        expected: '5 threads (15 annotations)',
-      },
-      {
-        rootThread: { totalChildren: 0, replyCount: 0 },
-        expected: 'No results',
-      },
-      {
-        rootThread: { totalChildren: 0, replyCount: 15 },
-        expected: 'No results',
-      },
-      {
-        rootThread: { totalChildren: 1, replyCount: 1 },
-        expected: '1 thread (1 annotation)',
-      },
-    ].forEach(test => {
-      it('renders number of threads and annotations', () => {
-        fakeUseRootThread.returns(test.rootThread);
-        const wrapper = createComponent();
-
-        assert.equal(
-          wrapper.find('.notebook-view__results').text(),
-          test.expected
-        );
-      });
-    });
+  it('renders results (counts)', () => {
```

Comment:
Looks like I missed a test for rendering filters, if we care.

Reply:
Adding a test.

Reply:
I generally avoid tests for parts of the output which are completely static (ie. is not conditionally rendered and doesn't contain any dynamic properties).